Clients.md

```diff
@@ -0,0 +1,74 @@
+# Featured Client Requirements
+Authored by: ShowierData9978 <contact@showierdata.xyz> on 4/18/24 (CT).
+
+## Terms/Definitions
+
+- Us/We - The Meower team
+
+## Rules
+- Clients created for malicious intent will not be featured
+- Clients must have their own Privacy Policy
+- If your client implements any kind of plugin/addon interface, any plugins that are featured (or in the repo) are considered to be part of the client.
+- The Meower Team may, at it's sole discretion, choose whether or not a client should be featured
+- The Meower Team may feature a client despite not complying with this RFC
+
+## Guidelines
+
+### Design Practices
+
+- Cache chat and post data
+- Use Cloudlink to get updates to cached information
+- You must use V1 of the WebSocket Server
```

Comment:
This should be future-proofed, and shouldn't be a requirement. Change it to something like "You should try to use the latest available API versions."